exporter/splunkhecexporter/testdata/integration_tests_config.yaml

```diff
@@ -1,20 +1,11 @@
-# SPLUNK DATA
-SPLUNK_IMAGE : "splunk/splunk:9.1"
-HOST : "non"
-
-#credentials:
-USER : "admin"
-PASSWORD : "helloworld"
-
-# ports
-UI_PORT: "8000"
-HEC_PORT: "8088"
-MANAGEMENT_PORT: "8089"
-
-# indexes
-EVENT_INDEX : "sck-otel"
-METRIC_INDEX: "sck-metrics"
-TRACE_INDEX: "sck-traces"
-
-# hec tokens
-HEC_TOKEN : "00000000-0000-0000-0000-000000000000"
\ No newline at end of file
+HOST: localhost
+USER: admin
+PASSWORD: helloworld
+UI_PORT: "62025"
+HEC_PORT: "62023"
+MANAGEMENT_PORT: "62024"
+EVENT_INDEX: sck-otel
+METRIC_INDEX: sck-metrics
+TRACE_INDEX: sck-traces
+HEC_TOKEN: 00000000-0000-0000-0000-000000000000
+SPLUNK_IMAGE: splunk/splunk:9.1.2
```

Comment:
Out of ignorance, how does this relate to the title of "using an older Splunk build"? Does `9.1` end up pulling a newer version than `9.1.2`?

Reply:
yes, it picks 9.1.4.